additional_data_sources/siren-function-source/scripts/import_siren.py

```diff
@@ -0,0 +1,192 @@
+                    ][0]["etatAdministratifUniteLegale"],
+                    "nomUniteLegale": result["unitesLegales"][i]["periodesUniteLegale"][
+                        0
+                    ]["nomUniteLegale"],
+                    "denominationUniteLegale": result["unitesLegales"][i][
+                        "periodesUniteLegale"
+                    ][0]["denominationUniteLegale"],
+                    "categorieJuridiqueUniteLegale": result["unitesLegales"][i][
+                        "periodesUniteLegale"
+                    ][0]["categorieJuridiqueUniteLegale"],
+                    "activitePrincipaleUniteLegale": result["unitesLegales"][i][
+                        "periodesUniteLegale"
+                    ][0]["activitePrincipaleUniteLegale"],
+                    "changementCategorieJuridiqueUniteLegale": result["unitesLegales"][
+                        i
+                    ]["periodesUniteLegale"][0][
+                        "changementCategorieJuridiqueUniteLegale"
+                    ],
+                    "nomenclatureActivitePrincipaleUniteLegale": result[
+                        "unitesLegales"
+                    ][i]["periodesUniteLegale"][0][
+                        "nomenclatureActivitePrincipaleUniteLegale"
+                    ],
+                    "economieSocialeSolidaireUniteLegale": result["unitesLegales"][i][
+                        "periodesUniteLegale"
+                    ][0]["economieSocialeSolidaireUniteLegale"],
+                    "caractereEmployeurUniteLegale": result["unitesLegales"][i][
+                        "periodesUniteLegale"
+                    ][0]["caractereEmployeurUniteLegale"],
+                }
+            )
+        except:
+            siren_info_list.append(
+                {
+                    "siren": result["unitesLegales"][i]["siren"],
+                    "unite_legale": "Not found",
+                    "derniertraitement": "Not found",
+                    "dateCreationUniteLegale": "Not found",
+                    "identifiantAssociationUniteLegale": "Not found",
+                    "trancheEffectifsUniteLegale": "Not found",
+                    "anneeEffectifsUniteLegale": "Not found",
+                    "dateDernierTraitementUniteLegale": "Not found",
+                    "categorieEntreprise": "Not found",
+                    "etatAdministratifUniteLegale": "Not found",
+                    "nomUniteLegale": "Not found",
+                    "denominationUniteLegale": "Not found",
+                    "categorieJuridiqueUniteLegale": "Not found",
+                    "activitePrincipaleUniteLegale": "Not found",
+                    "changementCategorieJuridiqueUniteLegale": "Not found",
+                    "nomenclatureActivitePrincipaleUniteLegale": "Not found",
+                    "economieSocialeSolidaireUniteLegale": "Not found",
+                    "caractereEmployeurUniteLegale": "Not found",
```

Comment:
plutôt mettre des None.
Pour que dans BigQuery on puisse faire "is not null" etc etc

Reply:
Oui je suis d'accord. J'ai repris le format des DA mais c'est mieux avec none